python_modules/libraries/dagster-looker/dagster_looker/api/resource.py

```diff
@@ -66,11 +62,15 @@ def read_config(_self) -> SettingsConfig:
         return init40(config_settings=DagsterLookerApiSettings())
 
     @public
+    @deprecated(
+        breaking_version="1.9.0",
+        additional_warn_text="Use dagster_looker.load_looker_asset_specs instead",
+    )
     def build_defs(
         self,
         *,
         request_start_pdt_builds: Optional[Sequence[RequestStartPdtBuild]] = None,
-        dagster_looker_translator: Optional[DagsterLookerApiTranslator] = None,
```

Comment:
For backcompat, since we're deprecating, I wonder if we should just pass `dagster_looker_translator.__class__` (so this change doesn't break existing users)